### PR TITLE
Ecommerce-Pricing,Invoice-1 to 4

### PR DESCRIPTION
**Pull request by devanshiwebiots**

### CODE REVIEW

File: package.json
Comment (devanshiwebiots) [inline package.json:1]: Inline styling used in whole Invoice section because it is like that in html and berlin theme

Comment (webiotsTanu) [reply package.json:1]: okay

File: src/Components/Applications/Ecommerce/Invoice/Invoice1/Invoice1Content.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { AmountDus, BilledTo, InvoiceDate, InvoiceNumber, KabulMatchettVandelayGroupLTD, UnitedStateTitleText } from "@/Constant";
+import { useAppSelector } from "@/Redux/Hooks";
+import { getallCardTotal } from "@/Utils/EcommerceService";
+
+const Invoice1Content = () => {
+  const { cart, tax } = useAppSelector((state) => state.cartData);
+
+  return (
+    <td>
+      <table className="bill-content" style={{ width: "100%" }}>
```
Comment (webiotsTanu) [inline src/Components/Applications/Ecommerce/Invoice/Invoice1/Invoice1Content.tsx]: reacstrap table should be used

Comment (devanshiwebiots) [reply src/Components/Applications/Ecommerce/Invoice/Invoice1/Invoice1Content.tsx]: table class is not required and if i add it using reactstrap then css not coming due to inline styling.Shall i change ?

Comment (webiotsTanu) [reply src/Components/Applications/Ecommerce/Invoice/Invoice1/Invoice1Content.tsx]: yes changed it with reactsrap

Comment (devanshiwebiots) [reply src/Components/Applications/Ecommerce/Invoice/Invoice1/Invoice1Content.tsx]: Okay

Comment (devanshiwebiots) [reply src/Components/Applications/Ecommerce/Invoice/Invoice1/Invoice1Content.tsx]: done

**Review by webiotsTanu — APPROVED** [review]

**Pull request merged**